assets/tests/consolidate_1_1_tests.yaml

```diff
@@ -2087,7 +2899,7 @@ tests:
             relations
               define parent: [folder] as self
               define viewer as viewer from parent
-        assertions:
+        checkAssertions:
```

Comment:
i see there's a few test cases where we haven't added `listAssertions`. Can you add assertions for the test cases involving contextual tuples? Since this code is not well covered with tests:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/5374887/223002635-27d4c68d-78b3-43ec-b50c-af6c85f7da2a.png">

Reply:
Thanks for the note - I will add them in subsequent PR (this one is large enough as is :) )